pkg/bosh/manifest/kube_converter.go

```diff
@@ -889,3 +861,28 @@ func (m *Manifest) ApplyBPMInfo(kubeConfig *KubeConfig, allResolvedProperties ma
 	}
 	return nil
 }
+
+// JobSpecCopierContainer will return a v1.Container{} with the populated field
+// TODO: fix inContainerReleasePath path
+func (m *Manifest) JobSpecCopierContainer(pathTODO string, containerName string, releaseName string, releaseImage string, volumeMountName string, containerMountPath string) v1.Container {
+
+	inContainerReleasePath := filepath.Join(pathTODO, releaseName)
+	// inContainerReleasePath := "/var/vcap/all-releases/jobs-src"
```

Comment:
Can we remove the commented code, it's probably part of the Git history if we need it again.

Reply:
sorry, I was testing something, did not expect you to review asap. This will become obsolete with my next change.